inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -279,28 +279,26 @@ void MKLDNNGraph::Replicate(const CNNNetwork &network, const MKLDNNExtensionMana
             graphEdges.push_back(edge);
         }
 
-//        for (auto &out_data : layer->outData) {
-//            if (getInputTo(out_data).empty()) {
-//                unused_data.insert(out_data);
-//            }
-//        }
+        if (op->get_type_info() != ngraph::op::v0::Result::type_info) {
```

Comment:
Do we really need this condition? I would expect Result::get_output_size() will always return 0

Reply:
Actually it returns 1.